src/qtcore/qml/UpdateGeometry.js

```diff
@@ -95,6 +95,8 @@ function updateHGeometry(newVal, oldVal, propName) {
         this.width = width;
 
     this.$updatingGeometry = false;
+
+    if (this.parent != undefined) updateChildrenRect(this.parent);
```

Comment:
This is the same as `if (this.parent)`. It should be either that or `if (this.parent !== undefined)`.